app/helpers/format-display-value.js

```diff
@@ -0,0 +1,31 @@
+import Ember from "ember";
+
+export default Ember.Helper.extend({
+  store: Ember.inject.service(),
+
+  getRecordName(modelName, modelId) {
```

Comment:
I'm not a big fan of having a helper lookup the store for records, but you can pass the model itself rather than the ID and do a `.get` directly on it. That'd be preferable.

Also, this looks way too specific to be a helper. Maybe this mapping should take effect when you build the normalized records (in the grouping code)

like

```
results[groupKey].actions.push(normalize(action));
```